package.json

```diff
@@ -7,37 +7,37 @@
     "url": "https://github.com/heroku/heroku-api-plugin/issues"
   },
   "engines": {
-    "node": ">= 16.0"
+    "node": ">=20"
```

Comment:
I guess this is right now. Probably should be requiring node 20 across plugins

Reply:
When I required 20 it failed the Node 22 tests.

Reply:
Right, I didn't mean to suggest doing something different. Just noting that we should be requiring 20 at a minimum now with plugins. Even if customers were to try to get this plugin to work with an older version of the cli that supported node 16, it wouldn't work.